snap/snapcraft.yaml

```diff
@@ -816,7 +816,7 @@ parts:
     source-depth: 1
     # Set source-branch to tag so that depth of 1 is relative to that tag.
     # This is equivalent to "git clone --depth 1 --branch <tag> <url>"
-    source-branch: v2.1.0
+    source-tag: v2.2.0-dev.7
```

Comment:
See comment above the source-branch: `Set source-branch to tag so that depth of 1 is relative to that tag.`

I think we should continue using `source-branch` to have reliable builds in case the tag is on a branch other than main. When using `source-tag`, the depth is relative to the main branch which may not actually include a commit with that tag.